README.md

```diff
@@ -77,6 +77,8 @@ via [Open Collective](https://opencollective.com/mui-org) or via [Patreon](http
   <a href="https://megafamous.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="https://mui.com/static/sponsors/megafamous.png" alt="megafamous.com" title="MegaFamous: Buy Instagram followers and likes." loading="lazy" /></a>
   <a href="https://www.dialmycalls.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="https://images.opencollective.com/dialmycalls/f5ae9ab/avatar/288.png" alt="dialmycalls.com" title="DialMyCalls: Send text messages, calls, and emails." loading="lazy" /></a>
   <a href="https://goread.io/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="https://images.opencollective.com/goread_io/eb6337d/logo/288.png" alt="goread.io" title="Goread.io: Instagram followers, likes, views, and comments." loading="lazy" /></a>
+  <a href="https://buzzoid.com/buy-instagram-followers/" rel="noopener sponsored" target="_blank"><img height="96" width="96" src="/static/sponsors/buzzoid-square.svg" alt="buzzoid.com" title="Buzzoid: Instant delivery Instagram followers" loading="lazy" /></a>
```

Comment:
I think this goes against the purpose: https://www.notion.so/mui-org/sales-Introduce-backlink-sponsor-plan-49e70924423e462aad099df16273c094?pvs=4#9f00df515854496697e45df38e913c73

```suggestion
```

or in a mini format.

Reply:
I did like the mini logo on the Figma, but happy to remove it from the readme - again, no strong feelings.

Reply:
I propose we try without, we see, best case scenario, we save time.